schema/bom-1.6.proto

```diff
@@ -652,24 +652,26 @@ enum Aggregate {
+  // The information is incomplete. Additional relationships exist and may include constituent components, services, or dependencies.
   AGGREGATE_INCOMPLETE = 2;
-  // The relationship is incomplete. Only relationships for first-party components, services, or their dependencies are represented.
+  // The information is incomplete. Only relationships for first-party components, services, or their dependencies are represented.
   AGGREGATE_INCOMPLETE_FIRST_PARTY_ONLY = 3;
-  // The relationship is incomplete. Only relationships for third-party components, services, or their dependencies are represented.
+  // The information is incomplete. Only relationships for third-party components, services, or their dependencies are represented.
   AGGREGATE_INCOMPLETE_THIRD_PARTY_ONLY = 4;
-  // The relationship may be complete or incomplete. This usually signifies a 'best-effort' to obtain constituent components, services, or dependencies but the completeness is inconclusive.
+  // The information may be complete or incomplete. This usually signifies a 'best-effort' to obtain constituent components, services, or dependencies but the completeness is inconclusive.
   AGGREGATE_UNKNOWN = 5;
-  // The relationship is incomplete. Only relationships for first-party components, services, or their dependencies are represented, limited specifically to those that are proprietary.
+  // The information is incomplete. Only relationships for first-party components, services, or their dependencies are represented, limited specifically to those that are proprietary.
   AGGREGATE_INCOMPLETE_FIRST_PARTY_PROPRIETARY_ONLY = 6;
-  // The relationship is incomplete. Only relationships for first-party components, services, or their dependencies are represented, limited specifically to those that are opensource.
+  // The information is incomplete. Only relationships for first-party components, services, or their dependencies are represented, limited specifically to those that are opensource.
   AGGREGATE_INCOMPLETE_FIRST_PARTY_OPENSOURCE_ONLY = 7;
-  // The relationship is incomplete. Only relationships for third-party components, services, or their dependencies are represented, limited specifically to those that are proprietary.
+  // The information is incomplete. Only relationships for third-party components, services, or their dependencies are represented, limited specifically to those that are proprietary.
   AGGREGATE_INCOMPLETE_THIRD_PARTY_PROPRIETARY_ONLY = 8;
-  // The relationship is incomplete. Only relationships for third-party components, services, or their dependencies are represented, limited specifically to those that are opensource.
+  // The information is incomplete. Only relationships for third-party components, services, or their dependencies are represented, limited specifically to those that are opensource.
   AGGREGATE_INCOMPLETE_THIRD_PARTY_OPENSOURCE_ONLY = 9;
+  // The information has been redacted.
+  AGGREGATE_REDACTED = 10;
```

Comment:
Redaction could happen only on first-party components while third-party information might be complete